sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -969,4 +970,26 @@ class JoinSuite extends QueryTest with SharedSQLContext {
           Seq(Row(0.0d, 0.0/0.0)))))
     }
   }
+
+  test("SPARK-28323: PythonUDF should be able to use in join condition") {
```

Comment:
Nice. can we add an assume too? `assume(shouldTestScalarPandasUDFs)`

Reply:
Ok.